### PR TITLE
Linh/Pool Tables (all 3) + Pool Charts

### LINKED ISSUE #1446: [TASK] Integrate back-end with Pool Transactions Table in Hubble Stats
Implement functionality to dynamically fetch and display the data for the Pool Transactions Table component on Hubble Stats.

Image

### LINKED ISSUE #1483: [TASK] Integrate back-end with Network Tables in Hubble Stats
Implement functionality to dynamically fetch and display the data for the Network Tables component on Hubble Stats.

- [ ] Network Pool Table
Image

- [ ] Network Token Table
Image

### LINKED ISSUE #1583: [TASK] Enhance Pool Details Page for Clearer Data Presentation
## Description
Based on discussion thread, the following updates are required to improve pool details page:

- Add a Volumes chart identical to the one on the overview page to pool details page, showing both deposit and withdrawal volumes.

- Revise Pool Data Table: Replace the TVL tab with tabs for 24h deposits and 24h withdrawals tabs to better reflect the pool's privacy status across chains within the latest time frame.

- For Relayer Earnings and Wrapped Fees, use historical data and add a footnote for clarification (e.g., '*historical relayer earnings').

### Screenshot
Image

### LINKED ISSUE #1585: [TASK] Integrate back-end with Pool Charts on Hubble Stats
Implement functionality to dynamically fetch and display the data for the Pool Charts component on Hubble Stats.

Image

### PR DESCRIPTION
**Pull request by vutuanlinh2k2**
## Summary of changes
_Provide a detailed description of proposed changes._
- Connect back-end with Pool Overview and Pool Wrapping Table
- Update new design for the detail page
- Clearer component names 
- Update suffix for charts and tables

### Proposed area of change
_Put an `x` in the boxes that apply._

- [ ] `apps/bridge-dapp`
- [x] `apps/hubble-stats`
- [ ] `apps/stats-dapp`
- [ ] `apps/webbsite`
- [ ] `apps/faucet`
- [ ] `apps/tangle-website`
- [ ] `libs/webb-ui-components`

### Reference issue to close (if applicable)
_Specify any issues that can be closed from these changes (e.g. Closes #233)._
- Closes #1583 
- Closes #1483 
- Closes #1446 
- Closes #1585 

### Screen Recording
_If possible provide a screen recording of proposed change._

https://github.com/webb-tools/webb-dapp/assets/69841784/64d4ffa5-961e-4a49-9b4a-6323032bb06a

-----

### Code Checklist 
_Please be sure to add .stories documentation if any additions are made to `libs/webb-ui-components`._

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Review by monaiuu — COMMENTED** [review]
Comments:

- Inconsistent chip color for Tangle Testnet. Please address to ensure background and icon is the same color.
- Unable to differenciate between the various orbit chains, can we use Athena, Hermes etc. for the chip texts?
Screenshot 2023-09-06 at 2 10 07 PM

- Not sure if `*Historical TWL` makes sense, can we update the copy to `Total Wrapped Locked?` instead?

**Review by AtelyPham — APPROVED** [review]

**Review by AtelyPham — APPROVED** [review]
LGTM 👍! Please wait for @monaiuu to review the UI before merging. Great work on this, @vutuanlinh2k2.

**Comment by monaiuu** [comment]
### Pre-Release Final Updates:

- [ ] Resolve issue where UI reverts to light mode upon refresh (see video).

https://github.com/webb-tools/webb-dapp/assets/109539270/60f771d9-092e-4228-aa39-c8049b947c24


- [ ] Standardize breadcrumb display to '`Webb Wrapped tTNT`' for consistency (note the lowercase 't').
Screenshot 2023-09-08 at 11 17 38 AM


- [ ] Update dark mode for the shielded chip component (see below).
Screenshot 2023-09-08 at 11 18 04 AM
Screenshot 2023-09-08 at 11 48 09 AM


- [ ] Enable pool page to open when selecting 'Webb Wrapped tTNT' in the shielded pools tab (currently non-functional).
Screenshot 2023-09-08 at 11 29 52 AM
Screenshot 2023-09-08 at 11 29 17 AM

- [ ] Eliminate excessive spacing in below tooltip.
Screenshot 2023-09-08 at 11 30 49 AM

- [ ] For mobile view, increase the font size of Tabs to match desktop view (`heading/05/bold`). Currently, tabs texts are too small.
Screenshot 2023-09-08 at 11 49 59 AM




- [ ] Ensure the footer component is anchored to the bottom. (see below for large screen)
Screenshot 2023-09-08 at 11 28 12 AM

**Pull request merged**
